ios/control_center_module-11up/@@PROJECTNAME@@.h

```diff
@@ -0,0 +1,7 @@
+#import <ControlCenterUIKit/CCUIToggleModule.h>
+
+@interface @@PROJECTNAME@@ : CCUIToggleModule {
+    BOOL _selected;
```

Comment:
Would it be simpler code if it was a property instead of an ivar and we only override the setter?

Reply:
I will see about that.